README.bn.adoc

```diff
@@ -67,18 +67,59 @@ toc::[]
   ** চন্দ্রবিন্দুর অবস্থান সংশোধন।
 
 
-== ইনস্টল
+== ইনস্টল  পদ্ধতি
 
-WARNING: ওপেনবাংলা কিবোর্ড ১.২.০ অথবা পূর্ববর্তী সংস্করণ ইনস্টল করা থাকলে সেটি প্রথমে আনইনস্টল করে নিন।
+IMPORTANT: ওপেনবাংলা কিবোর্ড ১.২.০ অথবা পূর্ববর্তী সংস্করণ ইনস্টল করা থাকলে সেটি প্রথমে আনইনস্টল করে নিন।
 
-ইনস্টলের সময় **কার্যকর ইন্টারনেট সংযোগ** প্রয়োজন।
+প্রথমে আপনার ডিস্ট্রোর জন্য আমাদের রিপজিটরি যোগ করুন, তারপর  আপনার প্যাকেজ ম্যানেজার দিয়ে স্বাভাবিকভাবে ইনস্টল করতে পারবেন।
 
-ওপেনবাংলা কিবোর্ড ইন্সটল করার জন্য টার্মিনাল উইন্ডো খুলে এই কমান্ডটি রান করুন:
-```bash
-wget -nv -O- https://bit.ly/OBK-sh | bash
+=== উবুন্টু এবং উবুন্টু-ভিত্তিক ডিস্ট্রো
+নিচের কমান্ডগুলো দিয়ে আমাদের রিপজিটরি যোগ করে নিন:
+```
+DIST=YOUR_DISTNAME
+curl https://dl.bintray.com/openbangla/i/pub.key | sudo apt-key add -
+curl https://dl.bintray.com/openbangla/i/ubuntu.conf | sed "s/@NAME@/$DIST/" | sudo tee /etc/sources.list.d/openbangla.list
+sudo apt-get update
+```
+WARNING: এখানে `YOUR_DISTNAME` এর জায়গায় আপনার ডিস্ট্রোর _বেজ উবুন্টু কোডনেম_ বসাতে হবে, যেমন: `bionic`, `focal`, ইত্যাদি।
```

Comment:
Can't we automate this `DISTRO_NAME`? I think many users will take it as a hassle. :disappointed:

Reply:
yeaaaaaah... i thought so. i'll have to test a few common repos.

Reply:
pushed 304390d9714544a2a10b8ff473f8f5539975a9da
bdeshi/openbangla-keyboard/issues/2 has to be done as well to finish up.

Reply:
@mominul **Done!** ([stedolan/jq](/stedolan/jq) is awesome)

i made a test deployment as well from my [`testing`](https://github.com/bdeshi/openbangla-keyboard/compare/testing) branch.
+ workflow run: https://github.com/bdeshi/openbangla-keyboard/actions/runs/40553940
+ deployed repo: https://bintray.com/openbangla-testing
+ deployed release: https://github.com/bdeshi/openbangla-keyboard/releases/tag/1.5.2

added another secret key, `BINTRAY_USER` for holding bintray username.

Reply:
@mominul: **Done** after all! ([stedolan/jq](/stedolan/jq) is awesome)

i made a test deployment from my [`testing`](https://github.com/bdeshi/openbangla-keyboard/compare/testing) branch.

+ workflow run: https://github.com/bdeshi/openbangla-keyboard/runs/450297539
+ deployed repo: https://bintray.com/openbangla-testing
+ deployed release: https://github.com/bdeshi/openbangla-keyboard/releases/tag/1.5.2

other changes:
* added another secret key, `BINTRAY_USER` for holding bintray username.
* there were a couple minor typos in the readme.
* squashed a remaining ubuntu deploy bug
* ubuntu17 is unmaintained (official repos are archived), so dropped ubuntu16 as well.